ownCloudAppFramework/Branding/Branding.m

```diff
@@ -49,7 +49,7 @@ + (void)load
 
 				@"can-add-account" : @(YES),
 				@"can-edit-account" : @(YES),
-				@"enable-review-prompt" : @(NO)
+				@"enable-review-prompt" : @(YES)
```

Comment:
Default value for branded clients should be `NO`

Reply:
I assumed that this value would be used even if the app is not branded, but now saw that this assumption was wrong. I changed it back to `NO` / `false` and added a note to the documentation for this parameter.